apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/StaticLargeObjectApiLiveTest.java

```diff
@@ -117,6 +118,7 @@ public void testDeleteSinglePartObjectWithMultiPartDelete() throws Exception {
          assertThat(resp.deleted()).isZero();
          assertThat(resp.notFound()).isZero();
          assertThat(resp.errors()).hasSize(1);
+         api.getObjectApi(regionId, containerName).delete(objectName);
```

Comment:
@zack-shoylev Is it the intention that this only be called if the assertions succeed?